File: truezip-kernel/src/test/java/de/truezip/kernel/sl/FsManagerLocatorTest.java
/*
 * Copyright (C) 2005-2012 Schlichtherle IT Services.
 * All rights reserved. Use is subject to license terms.
 */
package de.truezip.kernel.sl;

import static org.junit.Assert.assertNotNull;
import org.junit.Test;

/**
 * @author Christian Schlichtherle
 */
public final class FsManagerLocatorTest {
    @Test
    public void testIsLocatable() {
        assertNotNull(FsManagerLocator.SINGLETON.get());
    }
}

File: src/test/scala/net/java/truevfs/kernel/impl/FsManagerLocatorTest.scala
/*
 * Copyright © 2005 - 2021 Schlichtherle IT Services.
 * All rights reserved. Use is subject to license terms.
 */
package net.java.truevfs.kernel.impl

import net.java.truevfs.kernel.spec.sl._
import org.scalatest.matchers.should.Matchers._
import org.scalatest.wordspec.AnyWordSpec

/** @author Christian Schlichtherle */
class FsManagerLocatorTest extends AnyWordSpec {

  "The file system manager locator singleton" should {
    "provide a default file system manager" in pendingUntilFixed {
      FsManagerLocator.SINGLETON.get.isInstanceOf[DefaultManager] shouldBe true
    }
  }
}
